matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/aggregation/livelocation/DefaultLiveLocationAggregationProcessor.kt

```diff
@@ -66,11 +66,12 @@ internal class DefaultLiveLocationAggregationProcessor @Inject constructor() : L
         // Check if beacon info is outdated
         if (isBeaconInfoOutdated(beaconInfoContent, content)) {
             Timber.v("## LIVE LOCATION. Beacon info has timeout")
-            return
+            beaconInfoContent.hasTimedOut = true
+        } else {
+            // Update last location info of the beacon state event
```

Comment:
this comment doesn't add any extra information, is it needed?

Reply:
You are right, we can remove it since what we do here is clear.